programs/lib/flexpret_lock.c

```diff
@@ -12,7 +12,7 @@ void hwlock_acquire() {
  */
 void hwlock_release() {
     if (swap_csr(CSR_HWLOCK, 0) != 1) {
-        assert(false);
+        assert(false, ""); // FIXME: Magnus: Unsure what reason to write here
```

Comment:
This is a user fault. This means that the hwlock wasnt locked. So I think an exception is appropriate. Something like:

hwlock_release called but hwlock was not locked.